packages/mint-components/src/components/tax-and-cash/sqm-tax-and-cash-dashboard/sqm-tax-and-cash-dashboard.feature

```diff
@@ -84,11 +85,11 @@ Feature: Tax And Cash Dashboard
     Given the document has status <status>
     Then a danger alert indicating the <documentType> with a <taxAlertHeader> and <taxAlertMessage> appears
 
-    Examples: 
-      | status     | documentType | taxAlertHeader                                                             | taxAlertMessage                                                                                                                 |
-      | NOT_ACTIVE | W9           | Your W9 tax form has personal information that doesn't match your profile. | Please resubmit a new W9 form.                                                                                                  |
-      | NOT_ACTIVE | W8-BEN       | W8-BEN tax form is invalid.                                                | Your tax form may have expired or has personal information that doesn’t match your profile. Please submit a new W-8 BEN form.   |
-      | NOT_ACTIVE | W8-BEN-E     | W8-BEN tax form is invalid.                                                | Your tax form may have expired or has personal information that doesn’t match your profile. Please submit a new W-8 BEN-E form. |
+    Examples:
+      | status     | documentType | taxAlertHeader                                                              | taxAlertMessage                                                                                                                 |
+      | NOT_ACTIVE | W9           | Your W-9 tax form has personal information that doesn't match your profile. | Please resubmit a new W-9 form.                                                                                                 |
+      | NOT_ACTIVE | W8-BEN       | W-8 tax form is invalid.                                                    | Your tax form may have expired or has personal information that doesn’t match your profile. Please submit a new W-8 BEN form.   |
```

Comment:
@Locrian24 looking at the W-9 message, could be add 'Your' to the start of the sentence for consistency? This is  a nice to have, so if it requires more work (other than a quick update), let's leave it

Reply:
Just cause it's a prop on the component means it's a bit of a pain to update. Definitely doable but will need to be updated in a bunch of different places and add to the release scope a bit.

Reply:
nvm then, we do want to wrap the project up soon, probably don't have time for the change